site/en/docs/webstore/publish/index.md

```diff
@@ -53,18 +53,57 @@ the ZIP file and manifest depend on the specifics of your item.
   ZIP file in a way that reduces the size of the user download package. For details, see [Reducing
   the size of the user download package][9].
 
-Before you upload your item, you'll be asked to pick a developer account to own your items. Instead
-of your personal account, we suggest using a dedicated account.
-
-## Create a developer account {: #create-a-developer-account }
-
-Before you can publish on the Chrome Web Store, you need to [register as a Chrome Web Store
-developer][10]. We suggest using a new account just for your item instead of your personal account.
-
-If you already host your item in Google Play and you want your Chrome Web Store listing to show an
-"Available for Android" link, your item must have the same name as your Google Play listing, and
-both items must be owned by the same developer account. To transfer your Chrome Web Store item to a
-different developer, submit this [form][11].
+## Create and setup a developer account {: #create-setup-a-developer-account }
+
+Before you can publish your item on the Chrome Web Store, you'll be asked to pick a developer account to own your items and setup your developer account. 
+
+### Create your account {: #create-a-developer-account }
+
+First you will need to [register as a Chrome Web Store developer][10]. We suggest using a new account just for your item instead of your personal account.
+
+If you already host your item in Google Play and you want your Chrome Web Store listing to show an "Available for Android" link, your item must have the same name as your Google Play listing, and both items must be owned by the same developer account. To transfer your Chrome Web Store item to a different developer, submit this [form][11].
+
+### Setup your account {: #setup-a-developer-account }
+
+Once you've registered, you can finish setting up your developer account in the Account page located on the left menu.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/lBcTQm6QF1FBzOmLEfG1.png", alt="Chrome Web Store Account page", width="800", height="404" %}
+
+Here you can provide your developer profile information, configure management settings and enable email notifications. However, only the name, email and privacy policy link are mandatory.
+
+<table>
+  <thead>
+    <tr>
+      <th>Field</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Publisher name</strong></td>
+      <td>This name appears under the items' title. If you are a <a href="/docs/webstore/cws-dashboard-listing/#displaying-your-verified-publisher-status">verified publisher</a>, you can display an official publisher URL instead.
```

Comment:
In "under the items' title" it's unclear whether this is one item or multiple. (It implies a single title for multiple items.) Perhaps "under the title of each of your items" would be clearer?

Reply:
Done!